design/data_processing_operations.md

```diff
@@ -0,0 +1,121 @@
+# Data operations processing design
+
+## Document purpose
+
+Lay out goals / designs for KBase Collections data processing in the short, middle and long term.
+Said goals / designs will necessarly get more vague the further out in time we go.
+
+## Short term goals
```

Comment:
what timeframes are considered short term vs middle term?

Reply:
I was thinking

Short term = should be working on it now
Medium term = Most likely prior to RSV
Long term = post RSV, maybe by quite a while

Reply:
👍